src/Ouzo/Core/Logger/LoggerAdapter.php

```diff
@@ -91,6 +91,56 @@ public function log(string $level, $message, array $context = []): void
         }
     }
 
+    public function asLoggerInterface(): LoggerInterface
+    {
+        return new class() implements LoggerInterface {
+            public function log($level, $message, array $context = []): void
+            {
+                $this->log($level, $message, $context);
```

Comment:
Are you sure `$this` refers to the LoggerAdapter and not the anonymous class? We need test for it.